.eslintrc

```diff
@@ -76,8 +76,5 @@
         ]
       }
     ]
-  },
-  "parserOptions": {
-    "parser": "babel-eslint"
```

Comment:
das haben wir im frontend noch behalten

Reply:
Why should I keep it? We already have typescript-eslint